### PR TITLE
fixed broken links

### LINKED ISSUE #6256: Broken Links in documentation
**Describe the bug**
The link for raisinbread readme is broken in contributing.md

**To Reproduce**
Steps to reproduce the behavior (attach screenshots if applicable):
1. Go to `CONTRIBUTING.md`
2. Click on detailed steps link given under **Some things to keep in mind** (fifth part under the first point)
3. See error

**What did you expect to happen?**
RsaisinBread readme was supposed to open

**Additional context**
I am working on this issue

### PR DESCRIPTION
**Pull request by ridhishjain**
## Brief summary of changes

- [:+1: ] Have you updated related documentation?

#### Link(s) to related issue(s)

* Resolves #6256

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -62,7 +62,7 @@ issues and/or Redmine tickets (if applicable).
     4. For SQL patches that are _optional_ (e.g. those that perform some 
             cleanup), place them in `SQL/Cleanup_patches/`.
     5. The files for our test database (Raisinbread) must be updated if they were modified.
-        Detailed instructions for this step can be found [here](https://github.com/aces/Loris/tree/minor/raisinbread#exporting-rb).
+        Detailed instructions for this step can be found [here](https://github.com/aces/Loris/tree/master/raisinbread).
```
Comment (driusan) [inline CONTRIBUTING.md:65]: Since it's to a file in the repo, the link should be a relative link.

**Comment by driusan** [comment]
@ridhishjain sorry that it took so long to look at this because of the 23.0 release taking so much time. Would you be able to rebase this? It now has conflicts and can't be merged.

**Comment by ridhishjain** [comment]
@driusan resolved, can merge now.

**Comment by driusan** [comment]
After the merge the link seems to be going to the same place (except without the anchor). It looks like the link has been fixed as part of a different documentation overhaul.. I think we can close this. (I just tested on the GitHub frontend to confirm. the issue linked from the PR says it was fixed by #6546.)

**Pull request closed without merging**